platform/db/schema.rb

```
@@ -49,13 +62,46 @@
   end
 
+  create_table "pipeline_actions", force: :cascade do |t|
+    t.bigint "pipeline_id", null: false
+    t.bigint "query_action_id"
+    t.integer "action_type", null: false
+    t.integer "position", null: false
+    t.jsonb "action_data", default: {}, null: false
+    t.jsonb "result_data", default: {}, null: false
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["pipeline_id", "position"], name: "index_pipeline_actions_on_pipeline_id_and_position", unique: true
```

rubocop (1.69.1)

[convention] Use `%w` or `%W` for an array of words.

(Style/WordArray)